packages/markdown/processor/parse/flavored/embed.js

```diff
@@ -1,4 +1,4 @@
-const rgx = /^\[([\w ]*)\]\((\S*) ["'](@\w*)"\)/;
+const rgx = /^\[([^\]]*)\]\((\S*) ["'](@\w+)"\)/;
```

Comment:
What's this regex looking for?

Reply:
The whole pattern, or the `[^\]]` part specifically? Overall this just matches our flavored `@embed` link syntax:

   [Watch this video!](https://youtu.be/8bh238ekw3 "@embed")

The specific change means we'll match anything in the link title that isn't a closing `]` bracket (as opposed to only alphanumerics) so that this "field" permits for various grammars, etc!